mode/clike/clike.js

```diff
@@ -667,6 +667,9 @@ CodeMirror.defineMode("clike", function(config, parserConfig) {
         stream.eatWhile(/[\w\$_]/);
         return "meta";
       },
+      '*': function(stream) {
+        return stream.string.match(/^import.*/) ? 'variable' : 'operator';
```

Comment:
Maybe check whether `prevToken` is `"."` instead? This regexp would go wrong on a line like `import foo.*; return x *` (which I'm not even sure is valid Kotlin, but you get the point.)